src/Allegro.DotnetSdk/Allegro.DotnetSdk.csproj

```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.Build.NoTargets">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
-    <PackageVersion>2.1.1</PackageVersion>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
To jest bezpieczne prawda?

Reply:
Tak, zasadniczo jest ignorowane przez msbuild sdk resolver.